prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,71 @@ nav_section: prebid-mobile
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+## Requirements
+
+-   **Prebid Server Configuration**  
+    You *must* have a Prebid Server account in order to use Prebid Mobile. Prebid Server is a server-based host that communicates bid requests and responses between Prebid Mobile and demand partners.  
+
+    To set up your Prebid Server account for Prebid Mobile, refer to [Getting Started with Prebid Mobile]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+-   **Primary Ad Server Setup**  
+    Prebid Mobile enables you to retrieve bids simultaneously from multiple demand partners outside of a single ad server. This header bidding process does not determine a final winning bid. Ad ops users need to set up line items associated with each ad unit on a primary ad server. Prebid Mobile will send bids (via the primary ad server SDK) to the primary ad server. This ad server will include the bids from the header bidding process in determining a winning bid. (See "How It Works" below.)
+
+-   **Prebid Mobile SDK**  
+    Mobile app developers implement header bidding through the Prebid Mobile SDK integration. SDKs are available for [iOS](https://github.com/prebid/prebid-mobile-ios) and [Android](https://github.com/prebid/prebid-mobile-android).
+
+## How It Works
+
+The following diagram shows how the Prebid Mobile header bidding solution works.
 
 {: .pb-img.pb-lg-img :}
-![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/prebid-mobile.png)
+![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/pbm-overview-flow.png)
+
+1.  Prebid Mobile sends a request to Prebid Server.  
+    This request consists of the Prebid Server account ID and config ID for each tag included in the request.
+
+2.  Prebid Server constructs a bid request and passes it to the demand partners.  
+    An OpenRTB request is constructed for each demand partner from their Prebid Server adapter.
+
+3.  Each demand partner returns a bid response to Prebid Server.  
+    The bid response includes the bid price and the creative content.
+
+4.  Prebid Server sends the bid responses to Prebid Mobile.
+
+5.  Prebid Mobile sets key/value targeting for each ad slot through the primary ad server mobile SDK.  
+    This targeting will match the key/value targeting that was pre-configured on the prebid line items on the primary ad server.
```

Comment:
"This targeting will activate one or more of the Prebid line items that were previously configured in the primary ad server"